biogeochem/FatesSoilBGCFluxMod.F90

```diff
@@ -1126,6 +1146,102 @@ subroutine FluxIntoLitterPools(csite, bc_in, bc_out)
 
     end do  ! do elements
 
+    ! If we are coupled with MIMICS, then we need some assessment of litter quality
+    ! ie ligC/totalN.  If we are not tracking N in the litter flux (ie C-only model)
+    ! then we need to approximate this by estimating the mean C:N ratios of each
+    ! plant organ, and mulitplying that by the different C Fluxes to get a total
+    ! approximate N flux.  Note, in C-only, we will not capture any re-absorption.
+    
+    if(trim(hlm_decomp).eq.'MIMICS') then
+
+       ! If we track nitrogen (ie cnp or other) then
+       ! we diagnose the c-lig/n ratio directly from the pools
+       if(element_pos(nitrogen_element)>0) then
+
+          ! Sum totalN fluxes over depth [g/m2]
+          sum_N = sum((bc_out%litt_flux_cel_n_si(1:nlev_eff_soil) + &
+               bc_out%litt_flux_lig_n_si(1:nlev_eff_soil) + &
+               bc_out%litt_flux_lab_n_si(1:nlev_eff_soil)) * &
+               bc_in%dz_sisl(1:nlev_eff_soil))
+          
+       else
+          
+          ! In this case (Carbon Only), we use the stoichiometry parameters to estimate
+          ! the C:N of live vegetation and the seedbank, and use that
+          ! as a proxy for the C:N of the litter flux
+
```

Comment:
Thanks for checking in on this, your assessment is correct.  As you mention, we could add in resorption, but it will make things complicated.  On the bright side, when people use N cycling in FATES-CLM (via parteh and fates_parteh_mode=2) the re-absorption effects are included.